_data/team/roles/non_core_library_maintenance.yml

```
@@ -0,0 +1,84 @@
+role: Non-core library maintenance
+description: |
+  The non-core library maintenance team is responsible for
+  managing other packages that are directly
+  maintained by the MDAnalysis organization.
+
+tasks:
+  - Managing issues and pull-requests for sub-projects
+  - Managing new features and updates
+  - Managing maintenance
+  - Working with CI and CD team and infrastructure
+subroles:
+
+  - subrole: distopia
+    current_leads: [Richard Gowers,Hugo MacDermott-Opeskin]
+    current_members: [Richard Gowers,Hugo MacDermott-Opeskin, Rocco Meli]
+    historical_members:
+      - Richard Gowers
+      - Hugo MacDermott-Opeskin
+    
+
+  - subrole: cookiecutter-mdakit
+    current_leads:
+      - Lily Wang
+    current_members: []
+    historical_members:
+      - Lily Wang
+    
+
+  - subrole: mda-encore
+    current_leads: []
+    current_members: []
+    historical_members:
+      - Ian Kenney
+    
+
+  - subrole: hole2-mdakit
+    current_leads:
+      - Lily Wang
+    current_members: []
+    historical_members:
+      - Ian Kenney
+    
+
+  - subrole: benchmarks
+    current_leads:
+      - Oliver Beckstein
+    current_members: []
+    historical_members:
+      - Oliver Beckstein
+    
+
+  - subrole: GridDataFormats
+    current_leads:
+      - Oliver Beckstein
+    current_members: []
+    historical_members:
+      - Irfan Alibay
+      - Oliver Beckstein
+    
+  
+  - subrole: MDAnalysisData
+    current_leads:
+      - Oliver Beckstein
+    current_members: []
+    historical_members:
+      - Oliver Beckstein
+    
+  
+  - subrole: mda-xrlib and other small utilities
+    current_leads:
+      - Irfan Alibay
+    current_members: []
+    historical_members:
+      - Irfan Alibay
+    
+  
+  - subrole: pyedr
+    current_leads:
+      - Irfan Alibay
+    current_members: []
+    historical_members:
+      - Irfan Alibay
```

```suggestion
      - Irfan Alibay (2022-2023)
```
@jbarnoud should be here too